jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/DockerContextMojo.java

```diff
@@ -70,8 +70,7 @@ public void execute() throws MojoExecutionException {
 
       new JavaDockerContextGenerator(mavenProjectProperties.getJavaLayerConfigurations())
           .setBaseImage(getBaseImage())
-          .setJvmFlags(getJvmFlags())
-          .setMainClass(mainClass)
+          .setEntrypoint(getEntrypoint())
```

Comment:
I think this is missing the logic to handle JVM flags and main class? We might want to add some tests to make sure JVM flags and mainClass still work.